pkg/checksum/executor.go

```diff
@@ -132,6 +142,7 @@ func buildRequest(
 		if err != nil {
 			return nil, err
 		}
+		req.NotFillCache = true
```

Comment:
Please set it in `build(Table|Index)Request`.

Reply:
Oops... This line should be removed since `SetChecksumRequest` did this for us.